components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandler.java

```diff
@@ -250,6 +252,20 @@ public boolean validateScope(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
                 }
             }
         }
+        // Deriving the list of global level scope validation implementations.
```

Comment:
Need to add the logic to handle allowed scopes

Reply:
Added in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1468/commits/328d264a2e70a4028ddfcbf05a25f57e3e873174